pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,42 +14,62 @@
 
 package com.google.sps.servlets;
 
-import java.util.ArrayList;  
-import java.util.List;  
-import java.util.Date;  
-import com.google.gson.Gson;
+import java.util.ArrayList; 
+import java.util.Date; 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    
+    ArrayList<String> comments = new ArrayList<String>();  
```

Comment:
FYI you can omit the type argument on the constructor invocation and use the "diamond". Type will be inferred from the left side of the expression.

```java
ArrayList<String> comments = new ArrayList<>();
```

https://docs.oracle.com/javase/tutorial/java/generics/types.html#diamond

Reply:
Oh okay I changed it, thanks!